udata_front/theme/gouvfr/assets/js/components/dataset/resource/resource.vue

```diff
@@ -306,15 +336,44 @@ export default defineComponent({
     const owner = useOwnerName(props.resource);
     const resourceImage = useResourceImage(props.resource);
     const { getComponentsForHook } = useComponentsForHook();
+
     /** @type {import("vue").Ref<HTMLElement | undefined>} */
     const content = ref();
+
     const expanded = ref(false);
     const expand = () => {
+      if(expanded.value) {
+        globalThis._paq?.push(['trackEvent', 'navigation', 'Close resource', props.resource.id]);
+      } else {
+        globalThis._paq?.push(['trackEvent', 'navigation', 'Open resource', props.resource.id]);
+        if(hasExplore.value) {
+          registerEvent(resourcePreviewButtonId);
+        } else if (hasSchema.value) {
+          registerEvent(resourceStructureButtonId);
+        } else {
+          registerEvent(resourceInformationButtonId);
+        }
+      }
       expanded.value = !expanded.value;
       if(content.value) {
         toggleAccordion(content.value, expanded.value);
       }
     }
+
+    /**
+     *
+     * @param {import("vue").ComputedRef<string> | string} tab Tab name
+     */
+    const registerEvent = (tab) => {
+      const tabName = unref(tab);
+      globalThis._paq?.push(['trackEvent', 'navigation', 'View resource tab', props.resource.id, tab]);
```

Comment:
This one is mainly to have a general idea of the interactions on a given page (same as the other events in update-url, sort and tabs) because they aren't tracked otherwise.
While the 2 others are more related to our "impact".